source/streaming-mode/streaming-read-config.txt

```diff
@@ -46,6 +46,27 @@ You can configure the following properties when reading data from MongoDB in str
    * - ``collection``
```

Comment:
The following information provided in "Documentation Changes Summary" is not reflected anywhere:

> When scanning from multiple/all collections, collections can be freely dropped while being scanned.

Reply:
I moved the content from this table into [its own section here](https://preview-mongodbjordansmith721.gatsbyjs.io/spark-connector/DOCSP-36546-multiple-collections-support/streaming-mode/streaming-read-config/#specifying-multiple-collections-in-the-collection-property) and added that information

Reply:
This is resolved, but I don't have the technical ability to resolve the discussion.